sources/packages/backend/apps/queue-consumers/src/processors/schedulers/notification/process-notifications.scheduler.ts

```diff
@@ -0,0 +1,60 @@
+import { InjectQueue, Process, Processor } from "@nestjs/bull";
+import { Job, Queue } from "bull";
+import { NotificationService } from "@sims/services/notifications";
+import {
+  ProcessNotificationsQueueInDTO,
+  ProcessNotificationsResponseQueueOutDTO,
+} from "./models/notification.dto";
+import { BaseScheduler } from "../base-scheduler";
+import {
+  PROCESS_NOTIFICATIONS_POLLING_LIMIT,
+  PROCESS_NOTIFICATION_CLEANUP_PERIOD,
```

Comment:
mm...actually, we are saving these values in the qeueue_configuration table, I was under the impression that we will be taking the polling limit value from the table.
or are planning to add it to my PR? that also works, I added a method to retrieve the cleanup period

Reply:
Planning to leave it to your PR @ann-aot as the intention of this PR is to go hard-coded.